### PR TITLE
Updated dead link in "how-to-spend-company-money"

### PR DESCRIPTION
**Pull request by fmartingr**
#### Summary

Update the recommended books/resources for training link[^1] from the How to spend company money page since it does not exist.

[^1]: https://handbook.mattermost.com/operations/research-and-development/training

### CODE REVIEW

**Comment by justinegeffen** [comment]
Thank you! This link has just been updated so I've added the new link as a suggestion. Thanks for finding this page, I didn't realize it was out of date. :)

File: operations/finance/staff-member-expenses/how-to-spend-company-money.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ While we're sharing average costs for various items below, you should use your b
 * Laptop stand: Average price **180 USD**
 * Laptop carrying bag: Average price **60 USD**
 * Work-related books: **30 USD** per month or **360 USD** annually
-  * [Recommended resources](https://handbook.mattermost.com/operations/research-and-development/training)
```
Comment (justinegeffen) [inline operations/finance/staff-member-expenses/how-to-spend-company-money.md:44]: https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook/professional-development#additional-training-resources

**Comment by fmartingr** [comment]
> Thank you! This link has just been updated so I've added the new link as a suggestion. Thanks for finding this page, I didn't realize it was out of date. :)

I have updated the link with the correct one :)

**Review by justinegeffen — APPROVED** [review]
Thank you so much! :)

**Pull request merged**